packages/input/src/index.vue

```diff
@@ -243,7 +243,7 @@ export default defineComponent({
       resize: props.resize,
     }))
     const inputDisabled = computed(() => props.disabled || elForm.disabled)
-    const nativeInputValue = computed(() => String(props.modelValue))
+    const nativeInputValue = computed(() => props.modelValue === null || props.modelValue === undefined ? '' : String(props.modelValue))
```

Comment:
Is this intentional or it was a mistake? 
If you were trying to express with "if props.modelValue is nullish, then return empty string, otherwise return Stringified modelValue" then this line is wrong, you need a  parenthesis to wrap the expression